monkestation/code/modules/virology/reagents/vaccine.dm

```diff
@@ -10,7 +10,7 @@
 
 /datum/reagent/vaccine/on_mob_life(mob/living/carbon/drinker, seconds_per_tick, times_fired)
 	. = ..()
-	drinker.immune_system.ApplyVaccine(data["antigen"], 1, 30 MINUTES)
+	drinker.immune_system.ApplyVaccine(data["antigen"], 10, 30 MINUTES)
```

Comment:
i'd ask this is lowered to like 5, currently 25 units would be enough to cure most of the crew in most situations